django/db/backends/postgresql/introspection.py

```diff
@@ -203,13 +205,14 @@ def get_constraints(self, cursor, table_name):
                 LEFT JOIN pg_class c2 ON idx.indexrelid = c2.oid
                 LEFT JOIN pg_am am ON c2.relam = am.oid
                 LEFT JOIN pg_attribute attr ON attr.attrelid = c.oid AND attr.attnum = idx.key
-                WHERE c.relname = %s AND pg_catalog.pg_table_is_visible(c.oid)
+                WHERE c.relname = %%s AND pg_catalog.pg_table_is_visible(c.oid)
             ) s2
             GROUP BY indexname, indisunique, indisprimary, amname, exprdef, attoptions;
-        """, [table_name])
+        """ % {'index_name': self.index_name}, [table_name])
```

Comment:
Can we pass it in params?
```suggestion
        """, [self.index_name, table_name])
```